### PR TITLE
Options menu

### PR DESCRIPTION
**Pull request by Mir3605**
I've created the layout for the options menu. It is simple for now, it needs better graphics and can be developed in the future. For now, only the audio settings are functional, other tables are in a "to-do" state.
Screenshot_263

### CODE REVIEW

**Comment by Mir3605** [comment]
[Options Menu](https://trello.com/c/IgW0WQAD/26-options-menu)

File: Assets/MenuInterfaces/MainMenu.gd
Comment (ddziechciarz) [inline Assets/MenuInterfaces/MainMenu.gd:1]: Is changing scenes the best solution here? If it doesnt lag it's okay, but I believe showing/hiding elements on existing scene will be better performance wise

**Review by ddziechciarz — APPROVED** [review]
nice, but consider my comment before merging

File: Assets/MenuInterfaces/MainMenu.gd
Comment (Mir3605) [reply Assets/MenuInterfaces/MainMenu.gd:1]: I thought of it from the developer's perspective and decided that changing scenes gives us better code separation, simpler scenes, and consistent code for the buttons (as the other buttons also change scenes). I haven't noticed any performance issues with this solution, the transition seems to be instant.

**Pull request merged**